src/scss/_grid.scss

```diff
@@ -5,6 +5,9 @@
 	html,
 	body {
 		margin: 0;
+	}
+
+	body {
```

Comment:
Did having the background-color apply on the html element cause an issue?

Reply:
It does due to the negative z-index used on muted hero